start_integrate_test.sh

```diff
@@ -64,13 +64,17 @@ array+=("rpc/triple/msgpack")
 array+=("rpc/triple/pb/dubbogo-grpc")
 array+=("rpc/grpc")
 array+=("rpc/jsonrpc")
+array+=("rpc/triple/pb2")
 
 #tls
 array+=("tls/dubbo")
 array+=("tls/triple")
 array+=("tls/grpc")
 
 
+# replace tls config
+find $(pwd)/tls -type f -print0 | xargs -0 sed -i  's#\.\.\/\.\.\/\.\.#'"$(pwd)/tls"'#g'
```

Comment:
It is not a good idea to replace all of `../../..` with `$(pwd)/tls`, as this pattern may appear in other files. Therefore, I suggest replacing them one by one with a more specific pattern (including `x509`).

```bash
sed -i 's#../../../x509#'"$(pwd)/tls/x509"'#g' tls/dubbo/go-client/conf/dubbogo.yml
sed -i 's#../../../x509#'"$(pwd)/tls/x509"'#g' tls/dubbo/go-server/conf/dubbogo.yml
# etc..
```

Reply:
Furthermore, it would be better to print a message so that other users are aware of what we are doing.

```bash
echo "The certificate paths were replaced to "$(pwd)/tls/x509" for tls/dubbo/go-client/conf/dubbogo.yml, tls/dubbo/go-server/conf/dubbogo.yml, <others..>"
```

In order to make it more convenient to maintain, you could create an array to contain those config paths. Eventually, it looks like this:

```bash
tls_configs=("config path1" "config path2")
for (( i=0; i<${#tls_configs[@]}; i++ ))
do
  sed -i 's#../../../x509#'"$(pwd)/tls/x509"'#g' ${tls_configs[i]}
  echo "${tls_configs[i]}: The certificate paths were replaced to \"$(pwd)/tls/x509\""
done
```

Reply:
@justxuewei  we can only find yml type `find $(pwd)/tls -type f -name '*.yml' `

Reply:
Lgtm